__tests__/servertesting.test.js

```diff
@@ -20,6 +22,7 @@ describe("/get/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
+        body = body.topics;
```

Comment:
This could be done slightly more smoothly with destructuring e.g `{topics} = body`